airflow/models.py

```diff
@@ -711,126 +711,56 @@ def is_runnable(self, flag_upstream_failed=False):
         """
         return self.is_queueable(flag_upstream_failed) and not self.pool_full()
 
-    def are_dependents_done(self, main_session=None):
```

Comment:
Note for later: think of explaining that this has disapeared because now implemented as a regular Trigger